src/elfpy/utils/parse_config.py

```diff
@@ -91,16 +91,27 @@ def parse_simulation_config(config_file):
         amm=AMMConfig(**toml_config["amm"]),
         simulator=SimulatorConfig(**toml_config["simulator"]),
     )
-    match simulation_config.simulator.logging_level.lower():
-        case "debug":
-            level = logging.DEBUG
-        case "info":
-            level = logging.INFO
-        case "warning":
-            level = logging.WARNING
-        case "error":
-            level = logging.ERROR
-        case "critical":
-            level = logging.CRITICAL
+    
+    if simulation_config.simulator.logging_level.lower() == "debug":
```

Comment:
We updated the repo requirements to python 3.10 for the `match` statement below to work, among other reasons. I don't think the if statements are necessarily bad, but the match certainly look cleaner. Is the reason you changed because your python version is <3.10? I think it's better to keep the old version, but I'm open to discussing it.